lexos/helpers/general_functions.py

```diff
@@ -199,23 +200,34 @@ def apply_function_exclude_tags(input_string: str, functions: list) -> str:
     return striped_text
 
 
-def decode_bytes(raw_bytes: bytes) -> str:
+def decode_bytes(raw_bytes: bytes) -> bytes:
     """decode the raw bytes, typically used to decode `request.file`
 
     :param raw_bytes: the bytes you get and want to decode to string
     :return: A decoded string
     """
-    try:
-        # try to use utf-8 to decode first
-        encoding_type = "utf-8"
-        # Grab the file contents, which were encoded/decoded automatically
-        # into python's format
-        decoded_string = raw_bytes.decode(encoding_type)
-    except UnicodeDecodeError:
-        encoding_detect = chardet.detect(
-            raw_bytes[:constants.MIN_ENCODING_DETECT])  # Detect the encoding
-        encoding_type = encoding_detect['encoding']
-        # Grab the file contents, which were encoded/decoded automatically
-        # into python's format
-        decoded_string = raw_bytes.decode(encoding_type)
-    return decoded_string.replace("\x00", "")
+    bom16_le = str(codecs.BOM_UTF16_LE)
+    bom16_be = str(codecs.BOM_UTF16_BE)
+    if str(raw_bytes).startswith(bom16_le):
```

Comment:
no, no, no, the problem is that chardet is finding the wrong encoding, this solution will cause problem when you send in a more complicated string (for example an emoji with skin color information)